auslib/web/common/swagger/definitions.yml

```diff
@@ -338,6 +338,7 @@ definitions:
         format: ascii
         minLength: 0
         maxLength: 100
+        pattern: "(^[<>]?[=]?[0-9]*$)"
```

Comment:
This is close, but it still allows a few values it shouldn't, such as:
* =12
* <
* =

We should only allow plain numbers, or plain numbers prefixed by `<`, `>`, `<=`, or `>=`. A test such as:
```
diff --git a/auslib/test/admin/views/test_rules.py b/auslib/test/admin/views/test_rules.py
index 74643e05..25047abe 100644
--- a/auslib/test/admin/views/test_rules.py
+++ b/auslib/test/admin/views/test_rules.py
@@ -144,16 +144,23 @@ class TestRulesAPI_JSON(ViewTest):
         r = dbo.rules.t.select().where(dbo.rules.rule_id == rule_id).execute().fetchall()
         self.assertEqual(len(r), 1)
         self.assertEqual(r[0]["mapping"], "c")
         self.assertEqual(r[0]["backgroundRate"], 33)
         self.assertEqual(r[0]["priority"], 0)
         self.assertEqual(r[0]["memory"], "<7373")
         self.assertEqual(r[0]["data_version"], 1)
 
+    def testCreateRuleWithInvalidMemory(self):
+        for invalid in ("<", "=", "=12", "abc"):
+            ret = self._post(
+                "/rules", data=dict(backgroundRate=33, mapping="c", priority=0, memory=invalid, product="Firefox", update_type="minor", channel="nightly")
+            )
+            self.assertEqual(ret.status_code, 400, "does not match")
+
     def testCreateRuleWithMig64(self):
         ret = self._post("/rules", data=dict(backgroundRate=33, mapping="c", priority=0, mig64=True, product="Firefox", update_type="minor", channel="nightly"))
         rule_id = int(ret.get_data())
         self.assertEqual(ret.status_code, 200, "Status Code: %d, Data: %s" % (ret.status_code, rule_id))
         r = dbo.rules.t.select().where(dbo.rules.rule_id == rule_id).execute().fetchall()
         self.assertEqual(len(r), 1)
         self.assertEqual(r[0]["mapping"], "c")
         self.assertEqual(r[0]["backgroundRate"], 33)
```

...will help you ensure that you don't match any invalid values.